theme-contract/src/main/java/com/orange/ouds/theme/tokens/type/BorderStyle.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.theme.tokens.type
+
+import java.util.Locale
+
+enum class BorderStyle {
+    None, Solid, Dashed, Dotted;
+
+    companion object {
+        fun fromString(s: String): BorderStyle {
```

Comment:
`s` parameter could be renamed to `string` (for instance). Same comment for `e` below (there are other `e` in the code, I don't know why we did not replace them with `exception`).